### DISCUSSION TITLE
How to install CLI on Mac without homebrew?

### DISCUSSION THREAD

**Discussion by oelbaga** [stripe/stripe-cli#796] [Q&A]
I'm following the intstructions to install Stripe CLI on Mac without home-brew which it says to download and unzip the tar.gz file, but when that's done a terminal unix executable file entitled stripe is created and when you open it, it doesn't let you type anything. 

What am I missing here from the docs?

Instructions.
<img width="736" alt="Screen Shot 2021-12-05 at 11 26 14 AM" src="https://user-images.githubusercontent.com/11394862/144754816-9d36c8c7-f674-4868-bff4-8f701d4b5389.png">

Downloaded and unzipped files.
<img width="258" alt="Screen Shot 2021-12-05 at 11 27 30 AM" src="https://user-images.githubusercontent.com/11394862/144754824-c1e1d4b0-cc46-4f88-8632-45a86e111715.png">

After double clicking the Stripe Executable file.
<img width="567" alt="Screen Shot 2021-12-05 at 11 27 41 AM" src="https://user-images.githubusercontent.com/11394862/144754825-66da5a35-9da0-4328-9fd0-7ad26a5a4219.png">

**Comment by chetansharma10** [comment]
Install it in /usr/local/bin directory so that it is available and installed globally.

**Reply by ghost** [reply]
Worked for me, thanks!

**Comment by TheyCallMeFelix** [comment]
remember apple no longer supports 32bit here is the article: https://discussions.apple.com/thread/251478181?&previousThread=250777998021

Most importantly as of Ventura you need to set the default  from bash to zch shell here are the docs: 
https://support.apple.com/en-us/HT208050

happy hacking!
